### ISSUE TITLE
Translating an alert/recall: French text is read in English on the English page 

### ISSUE THREAD

**Issue by XiuJuanXia** [mmiron-HC/WST-RSAMS#80]
URL: https://hcdev.openplus.ca/en/documentation/translating-alertrecall
User Agent: Chrome, JAWS
Language: English

Issue Description: In the second paragraph, there is French text "fleur de lis" that is read in English.
![image](https://user-images.githubusercontent.com/88776404/129239009-d21cf10c-39ba-4dee-b10f-9858d4ed6a83.png)

Steps to reproduce: Navigate to the second paragraph and inspect the element. No lang="fr" found for the French text.

Expected results: French word "fleur de lis" should be read in French in the English page. lang="fr" should be added to the French text "fleur de lis" in the English page.

**Comment by ndolter** [comment]
Validated that the 'fr' lang attribute has been added and that the screen reader reads it correctly.
